src/Test/atomic_tests.jl

```diff
@@ -0,0 +1,244 @@
+"""
+    This function tests adding a single variable.
+"""
+function add_variable(model::MOI.ModelLike, config::TestConfig)
```

Comment:
It would be more consistent with the rest of the tests to call it `addvariabletest`